lib/acm_bot/bot.ex

```diff
@@ -12,12 +12,36 @@ defmodule AcmBot.Bot do
 
   command("start")
   command("armario")
+  command("junta")
 
   def handle({:command, :start, _message}, context) do
     answer(context, "Hola buenas tardes!\nEste es el nuevo bot de ACM.")
   end
 
-  # Versión precaria del comando armario. Es muy mejorable con las ideas de @IronJanoWar
+  def handle({:command, :junta, _message}, context) do
+    case AcmBot.Utils.get_board_info() do
+      {:ok, board_info} ->
+        board_message =
+          List.foldl(board_info, "", fn board_member, message ->
```

Comment:
This is a really weird way of doing this, first, why use `List.fold` instead of `Enum.reduce`?

Then, I think that it's better to do: `board_info |> Stream.map(&format_board_member/1) |> Enum.join("\n\n")`

Reply:
There's no reason to use `List.fold` over `Enum.reduce`, I'm just used to it.
About the `board_info |> Stream.map(&format_board_member/1) |> Enum.join("\n\n")` pipeline, you're totally right it's a lot cleaner that way, but does this means that `File.stream!` should be used too?

Reply:
I guess I can't use the `File.stream!` because I need the whole file to decode it, so in case `Jason` accepts a `Stream` as input to a `Jason.decode`, this is done at the very beginning of the pipeline so It wouldn't make much sense using `Stream` anyway.

Reply:
You should get used to the `Enum` methods over the `List` methods, just because they are much more generic :)

And you found out, but you don't need to have an stream, we use that stream there to not iterate twice over the enumerable